source/excerptexport/templates/excerptexport/templates/extraction_order_status.html

```diff
@@ -4,7 +4,8 @@
 
 {% block main %}
     <nav>
-        <a href="{% url 'excerptexport:index' %}">&#8810; Home</a>
```

Comment:
it might be a goo idea to consider making a breadcrumb instead of writing this everywhere. I'll file an issue, isn't urgent.